packages/channels/botpress-channel-messenger/src/messenger.js

```diff
@@ -214,303 +242,155 @@ class Messenger extends EventEmitter {
       })
   }
 
-  sendThreadRequest(body, method) {
-    return this.sendRequest(body, 'thread_settings', method)
+  sendPrivateReply(objectId, text, page_id) {
+    const token = this.getConfigVal('accessToken', page_id)
+    const url = `https://graph.facebook.com/v${this.config
+      .graphVersion}/${objectId}/private_replies?access_token=${token}`
+    return fetch(url, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        message: text
+      })
+    })
+      .then(this._handleFacebookResponse)
+      .then(res => res.json())
+      .catch(err => console.log(`Error posting private reply: ${err}`))
   }
 
-  sendTypingIndicator(recipientId, milliseconds) {
+  sendTypingIndicator(recipientId, milliseconds, page_id) {
```

Comment:
use this `sendTypingIndicator(recipientId, milliseconds, options)`

Reply:
Again, it will be confusing if some API calls have pageId at the end and some don't

If we want a full rewrite of API signatures perhaps this should be a different PR